composer.json

```diff
@@ -45,10 +34,22 @@
     },
     "bin": ["bin/coveralls"],
     "scripts": {
+        "install-devtools": [
+            "cd devtools && composer update"
```

Comment:
I was thinking about sth like that too. Then think about second composer file on same level, but that would override vendor (there could be deps only from one composer.json file).
That could be the good idea indeed.